python/nano/src/bigdl/nano/utils/inference/pytorch/model_utils.py

```diff
@@ -18,12 +18,14 @@
 import inspect
 from torch.utils.data import DataLoader
 import torch
+import pytorch_lightning as pl
 from bigdl.nano.utils.log4Error import invalidInputError
 
 
 def get_forward_args(model):
     forward_args = inspect.getfullargspec(model.forward).args[1:]
-    if isinstance(model, LightningModuleFromTorch):
+    if isinstance(model, LightningModuleFromTorch)\
+         or isinstance(model, FP32InferenceModelWrapped):
```

Comment:
to make it simple, we can have `LightningModuleFromTorch` and `FP32InferenceModelWrapped` to extend a common `nano_pytorch_wrapper` base?

Reply:
Since `FP32InferenceModelWrapped` is only an internal class while `LightningModuleFromTorch` is an external class, I think I will keep them separate.

Reply:
It doesn't matter if `FP32InferenceModelWrapped` is internal or external; the point is that if we introduce additional "model wrapper" classes later, the list can become longer; or we may need to check the condition in other places later. This can be hard to maintain.

To make it easy to maintain, we can have all these "model wrapper" classes to extend a common base class (`NanoPytorchWrapper`) that wraps a PyTorch model, and then just check `if isinstance(model, NanoPytorchWrapper)` in the code.